### PR TITLE
refactor(validation): Change config validation library

### PR DESCRIPTION
**Pull request by mattsjones**
I've made this change as the previous library doesn't support variant types for config which will likely be introduced in v8.

### CODE REVIEW

File: context/configSchema.js [hunk 1]
```diff
@@ -1,153 +1,121 @@
...
+    type: 'array',
+    items: {
+      type: 'object',
+      props: {
+        route: {
+          type: 'string'
+        },
+        name: {
+          type: 'string',
+          optional: true
```
Comment (michaeltaranto) [inline context/configSchema.js]: Is it a deliberate change to make `name` optional? Previously `entry` was the only optional key.

File: context/configSchema.js [hunk 2]
```diff
@@ -1,153 +1,121 @@
-const Schema = require('validate');
+const Validator = require('fastest-validator');
```
Comment (michaeltaranto) [inline context/configSchema.js]: sounds quick 😉

Comment (mattsjones) [reply context/configSchema.js]: Haha I know. The name is a bit lame but it was the only fit for purpose one I found.

File: context/configSchema.js (hunk 1 above)
Comment (mattsjones) [reply context/configSchema.js]: Yes, this was deliberate. It's technically not required and probably shouldn't have been in the first place.

**Pull request merged**